gym/tf_od_api/workspace/train_core/pre-trained-models/efficientdet_d0/pipeline.config

```diff
@@ -0,0 +1,191 @@
+            }
+          }
+          activation: SWISH
+          batch_norm {
+            decay: 0.9900000095367432
+            scale: true
+            epsilon: 0.0010000000474974513
+          }
+          force_use_bias: true
+        }
+        depth: 64
+        num_layers_before_predictor: 3
+        kernel_size: 3
+        class_prediction_bias_init: -4.599999904632568
+        use_depthwise: true
+      }
+    }
+    anchor_generator {
+      multiscale_anchor_generator {
+        min_level: 3
+        max_level: 7
+        anchor_scale: 4.0
+        aspect_ratios: 1.0
+        aspect_ratios: 2.0
+        aspect_ratios: 0.5
+        scales_per_octave: 3
+      }
+    }
+    post_processing {
+      batch_non_max_suppression {
+        score_threshold: 9.99999993922529e-09
+        iou_threshold: 0.5
+        max_detections_per_class: 100
+        max_total_detections: 100
+      }
+      score_converter: SIGMOID
+    }
+    normalize_loss_by_num_matches: true
+    loss {
+      localization_loss {
+        weighted_smooth_l1 {
+        }
+      }
+      classification_loss {
+        weighted_sigmoid_focal {
+          gamma: 1.5
+          alpha: 0.25
+        }
+      }
+      classification_weight: 1.0
+      localization_weight: 1.0
+    }
+    encode_background_as_zeros: true
+    normalize_loc_loss_by_codesize: true
+    inplace_batchnorm_update: true
+    freeze_batchnorm: false
+    add_background_class: false
+  }
+}
+train_config {
+  batch_size: 128
+  data_augmentation_options {
+    random_horizontal_flip {
+    }
+  }
+  data_augmentation_options {
+    random_scale_crop_and_pad_to_square {
+      output_size: 512
+      scale_min: 0.10000000149011612
+      scale_max: 2.0
+    }
+  }
+  sync_replicas: true
+  optimizer {
+    momentum_optimizer {
+      learning_rate {
+        cosine_decay_learning_rate {
+          learning_rate_base: 0.07999999821186066
+          total_steps: 300000
+          warmup_learning_rate: 0.0010000000474974513
+          warmup_steps: 2500
+        }
+      }
+      momentum_optimizer_value: 0.8999999761581421
+    }
+    use_moving_average: false
+  }
+  fine_tune_checkpoint: "PATH_TO_BE_CONFIGURED"
+  num_steps: 300000
+  startup_delay_steps: 0.0
+  replicas_to_aggregate: 8
+  max_number_of_boxes: 100
+  unpad_groundtruth_tensors: false
+  fine_tune_checkpoint_type: "classification"
+  use_bfloat16: true
+  fine_tune_checkpoint_version: V2
+}
+train_input_reader: {
+  label_map_path: "PATH_TO_BE_CONFIGURED/label_map.txt"
+  tf_record_input_reader {
+    input_path: "PATH_TO_BE_CONFIGURED/train2017-?????-of-00256.tfrecord"
+  }
+}
+
+eval_config: {
+  metrics_set: "coco_detection_metrics"
+  use_moving_averages: false
+  batch_size: 1;
+}
+
+eval_input_reader: {
+  label_map_path: "PATH_TO_BE_CONFIGURED/label_map.txt"
+  shuffle: false
+  num_epochs: 1
+  tf_record_input_reader {
+    input_path: "PATH_TO_BE_CONFIGURED/val2017-?????-of-00032.tfrecord"
```

Comment:
Are we leaving these `PATH_TO_BE_CONFIGURED` like this willingly, or should we change/remove these ones for clarity?

Reply:
I've just added  this pretrained  model, I didn't start work  with it  yet ... and about all other  comments and suggestions I can't  implement  them before  finishing the deployment  pipeline  first ... the model  deployment  pipeline  Indeed  needs improvements ... I will address  this issues as soon as possible  , thanks for commenting

Reply:
Ok, just wanted to confirm. Let's leave model specifications for later, then :)